presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5969,7 +5969,7 @@ public void testSelectColumnOfNulls()
     @Test
     public void testSelectCaseInsensitive()
     {
-        assertQuery("SELECT ORDERKEY FROM ORDERS");
+        assertQuery("SELECT ORDERKEY FROM orders");
```

Comment:
retract this change and add a comment that `testSelectCaseInsensitive` intentionally uses non-lowercased table name in SQL